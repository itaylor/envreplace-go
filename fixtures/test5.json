{
  "files": {
    "./test5.txt": ["../output/test5a.txt.out", "../output/test5b.txt.out"]
  }
}
