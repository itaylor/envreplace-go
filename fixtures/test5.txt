First I #FOO# then I #BAR#
