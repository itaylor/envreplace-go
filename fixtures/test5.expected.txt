First I test1 then I test2
